paasta_tools/chronos_serviceinit.py

```diff
@@ -30,24 +30,24 @@
 logging.basicConfig()
 
 
-# Calls the 'manual start' endpoint in Chronos (https://mesos.github.io/chronos/docs/api.html#manually-starting-a-job),
-# running the job now regardless of its 'schedule' and 'disabled' settings. The job's 'schedule' is left unmodified.
-def start_chronos_job(service, instance, job_id, client, cluster, job_config, emergency=False):
+def start_chronos_job(service, instance, job_id, client, cluster, complete_job_config, emergency=False):
+    """
+    Calls the 'manual start' Chronos endpoint (https://mesos.github.io/chronos/docs/api.html#manually-starting-a-job),
+    running the job now regardless of its 'schedule'. The job's "schedule" is unmodified.
+    """
     name = PaastaColors.cyan(job_id)
     log_reason = PaastaColors.red("EmergencyStart") if emergency else "Brutal bounce"
-    log_immediate_run = " and running it immediately" if not job_config["disabled"] else ""
+
     _log(
         service=service,
-        line="%s: Sending job %s to Chronos%s" % (log_reason, name, log_immediate_run),
+        line="%s: Sending job %s to Chronos" % (log_reason, name),
```

Comment:
This English sounds strange.

How about "%s: Kicking off manual run of %s in Chronos"